iceoryx_posh/include/iceoryx_posh/popo/untyped_publisher.hpp

```diff
@@ -37,9 +37,11 @@ class UntypedPublisherImpl : public base_publisher_t
     using base_publisher_t::getServiceDescription;
     using base_publisher_t::getUid;
     using base_publisher_t::hasSubscribers;
-    using base_publisher_t::isOffered;
-    using base_publisher_t::loan;
-    using base_publisher_t::loanPreviousSample;
+    using base_publisher_t::isOffered; // iox-#408 better hasOffered ?
```

Comment:
`hasOffered` is not better IMO

Reply:
`isOffering` or `doesOffer`?

Reply:
Just because the entity is a publisher, which either has or has not offered a topic. I can live with both but if we want to change this now is the time.

Reply:
I find it more intuitive to think about the present, not the past. So I would prefer a `is` or `does` prefix. Depending o the point of view, one could say the service of the publisher `isOffered` or the publisher `isOffering/doesOffer` a service

Reply:
> one could say the service of the publisher `isOffered`
This was my thinking, and I personally prefer it to "is offering".

Reply:
I leave it as is.